src/index.html

```diff
@@ -1,17 +1,151 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="block-distance">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          Surname:
+          <input type="text"
+                 name="surname"
+                 autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
Зверни увагу на форматування і відступи
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation